src/app/modules/funds/funds.service.ts

```diff
@@ -1,17 +1,30 @@
-import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs';
-import { URL } from '../../core/constants/api';
-import { Fund } from '../../shared/models/fund';
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable, Subject} from 'rxjs';
+
+import {URL} from '../../core/constants/api';
+import {Fund} from '../../shared/models/fund';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class FundsService {
-  constructor( private http: HttpClient ) {
-  }
+  private subject = new Subject<any>();
```

Comment:
Why do you use `any` type? Would it be better to have `Fund` instead `any` here?

Reply:
done